conjur-api/SearchVariableResult.cs

```diff
@@ -0,0 +1,63 @@
+﻿// <copyright file="VariableInfo.cs" company="Cyberark Inc.">
+//     Copyright (c) 2018 Cyberark Inc. All rights reserved.
+// </copyright>
+// <summary>
+//     Variable Info
+// </summary>
+
+
+namespace Conjur
+{
+    using System.Runtime.Serialization;
+
+    [DataContract]
+    public class SearchVariableResult
```

Comment:
These all should go on the `Resource` class instead, which `List` would instantiate with the data. Then a `Refresh()` method should be provided to refresh this info (or fetch it if missing).